src/wallet/isWalletASmartWallet.ts

```diff
@@ -0,0 +1,37 @@
+import { IsWalletASmartWalletOptions } from './types';
+import { Address, BlockTag, Hex, decodeAbiParameters } from 'viem';
```

Comment:
are those types? if so do an import for the types and one for the functions.

Reply:
Yes those are a mix of types and a function - will do!

Reply:
Updated to:

`import { decodeAbiParameters } from 'viem';
import type { Address, BlockTag, Hex } from 'viem';`